integration-tests/tests/aws/nino-check-unhappy.test.ts

```diff
@@ -0,0 +1,94 @@
+import {
+  clearItems,
+  populateNinoTable,
+} from "./resources/nino-check-dynamodb-helper";
+import { executeStepFunction } from "./resources/nino-check-stepfunction-helper";
+
+describe("Nino Check", () => {
+  const sessionId = "12345";
+  const nino = "AA000003D";
+  const badNino = "abc";
+  const testUser = {
+    nino: nino,
+    dob: "1948-04-23",
+    firstName: "Jim",
+    lastName: "Ferguson",
+  };
+
+  beforeEach(async () => {
+    await populateNinoTable(testUser);
+  });
+
+  afterEach(async () => {
+    await clearItems(process.env.NINO_USERS_TABLE as string, {
+      nino: testUser.nino,
+    });
+    await clearItems(process.env.NINO_ATTEMPTS_TABLE as string, {
+      id: sessionId,
+    });
+  });
+
+  describe("Unhappy User Input Paths", () => {
+    it("should fail when there is no sessionId present", async () => {
+      const startExecutionResult = await executeStepFunction({
+        nino: nino,
+      });
+      expect(startExecutionResult.output).toBe('{"sessionId":"12345"}');
```

Comment:
is this correct?

Reply:
Ok, so either this is wrong or the step function local version is wrong. I think the state machine needs to be checked to be sure

Reply:
I see what you mean now, my typo!